tests/listobjects/listobjects.go

```diff
@@ -176,7 +187,7 @@ func runTests(t *testing.T, schemaVersion string, client ListObjectsClientInterf
 							checkResp, err := client.Check(ctx, &pb.CheckRequest{
 								StoreId:          storeID,
 								TupleKey:         tuple.NewTupleKey(object, assertion.Request.Relation, assertion.Request.User),
-								ContextualTuples: assertion.Request.ContextualTuples,
+								ContextualTuples: contextualTuples,
 							})
 							require.NoError(t, err)
 							require.True(t, checkResp.Allowed, fmt.Sprintf("Expected Check(%s#%s@%s) to be true, got false", object, assertion.Request.Relation, assertion.Request.User))
```

Comment:
Adding this extra log message below isn't necessary, because the default output makes it clear that the expectation was true and not false. We're purposefully not using this field for that reason elsewhere, so to be consistent let's drop the verbose message here as well.